images/windows-hpc-base/cc0-legalcode.txt

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
```

Comment:
Shouldn't we just use the repo's ASLv2?

the resulting image on top of the base will be largely ASLv2 anyhow.

Reply:
The equivalent image that gets published to MCR was released with the CC0 license - https://github.com/microsoft/windows-host-process-containers-base-image/blob/main/cc0-license.txt

Reply:
I mean we're going to build ASLv2 images with this, and CC0 is NOT on the CNCF approved list so we should not be using any CC0 images.

Are we using CC0 anywhere else ...? 

https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md

Reply:
Correct.

Reply:
Also (I'm sure you saw mark) but the script and Makefile need the boilerplate repo headers, and the automation will look for the apache license and fail if we use a different one.